app/src/main/java/com/portfolio/tasky/globals/LoginUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.portfolio.tasky.globals
+
+class LoginUtils {
+
+    companion object{
+        var token : String = ""
+        fun setAccessToken(token : String){
+            this.token = token
+        }
+
+        fun getAccessToken() : String{
+            return "Bearer $token"
+        }
+    }
```

Comment:
Ah I now noticed that you don't have preferences yet. Not sure if using that was your plan or not, but to keep the user logged in, you'll need to store the token persistently. Using a companion object won't be enough.

Reply:
Removed this file

Done